<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="1.3 RNA Sequencing | _main.utf8" />
<meta property="og:type" content="book" />







<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>

<meta name="description" content="1.3 RNA Sequencing | _main.utf8">

<title>1.3 RNA Sequencing | _main.utf8</title>

<script src="libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="libs/navigation-1.1/tabsets.js"></script>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
</style>
</head>

<body>

<div class="container-fluid main-container">


<div class="row">
<div class="col-sm-12">
<div id="TOC">
<ul>
<li class="has-sub"><a href="index.html#intro"><span class="toc-section-number">1</span> Introduction</a><ul>
<li><a href="1-1-need-for-better-therapeutics.html#need-for-better-therapeutics"><span class="toc-section-number">1.1</span> Need for better therapeutics</a></li>
<li><a href="1-2-engineered-human-myocardium.html#engineered-human-myocardium"><span class="toc-section-number">1.2</span> Engineered Human Myocardium</a></li>
<li class="has-sub"><a href="1-3-rna-sequencing.html#rna-sequencing"><span class="toc-section-number">1.3</span> RNA Sequencing</a><ul>
<li><a href="1-3-rna-sequencing.html#bulk-rna-seq"><span class="toc-section-number">1.3.1</span> Bulk RNA Seq</a></li>
<li><a href="1-3-rna-sequencing.html#single-cell-rna-seq"><span class="toc-section-number">1.3.2</span> Single-cell RNA Seq</a></li>
</ul></li>
<li><a href="1-4-computational-deconvolution.html#computational-deconvolution"><span class="toc-section-number">1.4</span> Computational deconvolution</a></li>
<li><a href="1-5-principal-component-analysis-pca.html#principal-component-analysis-pca"><span class="toc-section-number">1.5</span> Principal Component Analysis (PCA)</a></li>
<li><a href="1-6-rationale-for-the-current-work.html#rationale-for-the-current-work"><span class="toc-section-number">1.6</span> Rationale for the current work</a></li>
</ul></li>
<li><a href="2-aims-and-objectives.html#aims-and-objectives"><span class="toc-section-number">2</span> Aims and Objectives</a></li>
<li><a href="3-materials-and-methods.html#materials-and-methods"><span class="toc-section-number">3</span> Materials and Methods</a></li>
<li><a href="4-results-and-discussion.html#results-and-discussion"><span class="toc-section-number">4</span> Results and Discussion</a></li>
<li><a href="5-conclusion-and-future-work.html#conclusion-and-future-work"><span class="toc-section-number">5</span> Conclusion and Future Work</a></li>
<li><a href="6-references.html#references"><span class="toc-section-number">6</span> References</a></li>
</ul>
</div>
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="rna-sequencing" class="section level2">
<h2><span class="header-section-number">1.3</span> RNA Sequencing</h2>
<p>Information stored in genes as DNA is transcribed into RNA and ultimately translated into proteins, this is the central dogma of biology. The transcription of a subset of genes into RNA molecules gives a cell it’s specificity and identity, along with regulating its activities. ‘Transcriptome’ is a term that refers to the total RNA, whether from a population of cells or a single cell. It is also used to refer to the total mRNA, which focuses on gene expression. Previously, gene expression studies relied on techniques such as northern blots and quantitative polymerase chain reaction  (qPCR) which are low-throughput and are limited to measuring a single transcript. The last two to three decades has seen the evolution of assays to quantify genome-wide gene expression, better known as transcriptomics. Microarrays, a hybridization based approach, were the main-stay of such transcriptomics until the recent revolution of high-throughput next-generation sequencing (NGS). It has revolutionized transcriptomics by enabling RNA analysis via the sequencing of complementary cDNA <span class="citation">(Wang, Gerstein, and Snyder <a href="#ref-wangRNASeqRevolutionaryTool2009">2009</a>)</span>. This is termed as RNA sequencing  (RNA-Seq) which has distinct advantages over the former approaches, such as its ability to detect transcripts that are not yet annotated, low background signal (in comparison to DNA microarrays), a large dynamic range of expression level, higher sensitivity, higher reproducibility, all of which allow for understanding the dynamic and complex nature of the transcriptome.The type of information that RNA-Seq provides can be broadly classified into two categories:
- Qualitative data which includes identifying transcripts, identifying intron/extron boundaries, poly-A sites and transcriptional start sites (TSS) which in RNA-Seq terminology is commonly referred to as “annotation”.
- Quantitative data which includes measuring differences in expression, alternative TSS, alternative splicing, alternative polyadenylation between two or more treatments or groups.
This power of sequencing RNA has led to RNA-Seq not only being limited to the genomics community but has also led to it becoming a main-stay in the toolkit of all life science research communities. A typical RNA-Seq experiment can be split into three parts <span class="citation">(Conesa et al. <a href="#ref-conesaSurveyBestPractices2016a">2016</a>)</span>:</p>
<ol style="list-style-type: decimal">
<li>Pre Analysis
<ul>
<li>Experimental Design (choosing the library type, sequencing legth, the number of replicates and sequencing depth)</li>
<li>Sequencing Design (spike-ins, randomization at library prep, randomization at sequencing run)</li>
<li>Quality Control (raw reads, read alignment, quantification, reproducibility)</li>
</ul></li>
<li>Core Analysis
<ul>
<li>Transcriptomic Profiling (read alignment, transcript discovery, quantification level, quantification measure)</li>
<li>Normalization (Z-scale, variance stabilized transformation, etc)</li>
<li>Differential Expression</li>
<li>Interpretation (functional profiling)</li>
</ul></li>
<li>Advanced Analysis
<ul>
<li>Visualization</li>
<li>Integration (eQTL, ATAC-seq, ChIP-Seq, proteomics/metabolomics)</li>
</ul></li>
</ol>
<p>The sucess of an RNA-Seq study depends on the choices and decisions made at each of these steps. Naturally, the real-world analysis of RNA-Seq data has as many variations as there are applications of the technology.</p>
<div id="bulk-rna-seq" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Bulk RNA Seq</h3>
<p>Generally, unless otherwise specified RNA-Seq refers to <strong>bulk</strong> RNA-Sequencing. Here, the RNA is collected from an entire tissue (biopsy), or a group of cells and thereby the sequenced data represents the <em>average expression level</em> for each gene across the large population of input. This bulk RNA-Seq which is the main work horse of gene expression studies is adequate for comparative transcriptomics, wherein samples of the same tissue are compared across species, or for quantifying expression signatures from ensembles, such as in disease studies. However, it falls short in its ability to be an effective tool for studying heterogeneous systems, such as complex tissues (brain, heart, etc) or early developmental studies. It also fails to capture the stochastic nature of gene expression and spatial resolution can not be obtained. An ilustrated, simplistic example is shown in <a href="1-3-rna-sequencing.html#fig:singleVsBulk">1.2</a>.</p>
<div class="figure"><span id="fig:singleVsBulk"></span>
<img src="data/bulkSingle.001.png" alt="A caption" width="100%" />
<p class="caption">
Figure 1.2: A caption
</p>
</div>
</div>
<div id="single-cell-rna-seq" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Single-cell RNA Seq</h3>
<p>Single-cell RNA-Seq  (scRNA-seq) is a new and active field of RNA-seq which arose to fulfill the unmet needs of bulk RNA-Seq. Despite being introduced in 2009 <span class="citation">(Tang et al. <a href="#ref-tangMRNASeqWholetranscriptomeAnalysis2009">2009</a>)</span>, it did not gain popularity until the advent of newer protocols and reduced sequencing costs much later. It measures the <em>distribution of expression levels</em> for each gene across a population of cells. It has revealed new, unknown cell types in what were considered to be well-studied and established diseases, such as the discovery of ionocyte cells, in cystic fibrosis <span class="citation">(Montoro et al. <a href="#ref-montoroRevisedAirwayEpithelial2018">2018</a>)</span>. Spatially resolved scRNA-Seq holds similar promises, revealing novel information on the extent of fetal marker gene expression in small populations of adult heart tissues <span class="citation">(pubmeddev and al et, <a href="#ref-pubmeddevSpatialDetectionFetal">n.d.</a>)</span>. Thus, novel biological questions addressing cell type identification, heterogeneity of cell responses, stochasticity of gene expression and inference of gene regulatory networks across cells can be studiesd. The applications of scRNA-Seq to novel biological questions and the computational and laboratory methods catering to it are advancing at such a rapid pace that even recent reviews <span class="citation">(Stegle, Teichmann, and Marioni <a href="#ref-stegleComputationalAnalyticalChallenges2015">2015</a>; Svensson, Vento-Tormo, and Teichmann <a href="#ref-svenssonExponentialScalingSinglecell2018">2018</a>)</span> are becoming outdated.</p>
</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-conesaSurveyBestPractices2016a">
<p>Conesa, Ana, Pedro Madrigal, Sonia Tarazona, David Gomez-Cabrero, Alejandra Cervera, Andrew McPherson, Michał Wojciech Szcześniak, et al. 2016. “A Survey of Best Practices for RNA-Seq Data Analysis.” <em>Genome Biology</em> 17 (1): 13. <a href="https://doi.org/10.1186/s13059-016-0881-8">https://doi.org/10.1186/s13059-016-0881-8</a>.</p>
</div>
<div id="ref-montoroRevisedAirwayEpithelial2018">
<p>Montoro, Daniel T., Adam L. Haber, Moshe Biton, Vladimir Vinarsky, Brian Lin, Susan E. Birket, Feng Yuan, et al. 2018. “A Revised Airway Epithelial Hierarchy Includes CFTR-Expressing Ionocytes.” <em>Nature</em> 560 (7718): 319–24. <a href="https://doi.org/10.1038/s41586-018-0393-7">https://doi.org/10.1038/s41586-018-0393-7</a>.</p>
</div>
<div id="ref-pubmeddevSpatialDetectionFetal">
<p>pubmeddev, and Asp M. al et. n.d. “Spatial Detection of Fetal Marker Genes Expressed at Low Level in Adult Human Heart Tissue. - PubMed - NCBI.” https://www.ncbi.nlm.nih.gov/pubmed/29021611?dopt=Abstract.</p>
</div>
<div id="ref-stegleComputationalAnalyticalChallenges2015">
<p>Stegle, Oliver, Sarah A. Teichmann, and John C. Marioni. 2015. “Computational and Analytical Challenges in Single-Cell Transcriptomics.” <em>Nature Reviews. Genetics</em> 16 (3): 133–45. <a href="https://doi.org/10.1038/nrg3833">https://doi.org/10.1038/nrg3833</a>.</p>
</div>
<div id="ref-svenssonExponentialScalingSinglecell2018">
<p>Svensson, Valentine, Roser Vento-Tormo, and Sarah A. Teichmann. 2018. “Exponential Scaling of Single-Cell RNA-Seq in the Past Decade.” <em>Nature Protocols</em> 13 (4): 599–604. <a href="https://doi.org/10.1038/nprot.2017.149">https://doi.org/10.1038/nprot.2017.149</a>.</p>
</div>
<div id="ref-tangMRNASeqWholetranscriptomeAnalysis2009">
<p>Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. “mRNA-Seq Whole-Transcriptome Analysis of a Single Cell.” <em>Nature Methods</em> 6 (5): 377–82. <a href="https://doi.org/10.1038/nmeth.1315">https://doi.org/10.1038/nmeth.1315</a>.</p>
</div>
<div id="ref-wangRNASeqRevolutionaryTool2009">
<p>Wang, Zhong, Mark Gerstein, and Michael Snyder. 2009. “RNA-Seq: A Revolutionary Tool for Transcriptomics.” <em>Nature Reviews. Genetics</em> 10 (1): 57–63. <a href="https://doi.org/10.1038/nrg2484">https://doi.org/10.1038/nrg2484</a>.</p>
</div>
</div>
<p style="text-align: center;">
<a href="1-2-engineered-human-myocardium.html"><button class="btn btn-default">Previous</button></a>
<a href="1-4-computational-deconvolution.html"><button class="btn btn-default">Next</button></a>
</p>
</div>
</div>


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
