<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="1.4 Computational deconvolution | _main.utf8" />
<meta property="og:type" content="book" />







<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>

<meta name="description" content="1.4 Computational deconvolution | _main.utf8">

<title>1.4 Computational deconvolution | _main.utf8</title>

<script src="libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="libs/navigation-1.1/tabsets.js"></script>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
</style>
</head>

<body>

<div class="container-fluid main-container">


<div class="row">
<div class="col-sm-12">
<div id="TOC">
<ul>
<li class="has-sub"><a href="index.html#intro"><span class="toc-section-number">1</span> Introduction</a><ul>
<li><a href="1-1-need-for-better-therapeutics.html#need-for-better-therapeutics"><span class="toc-section-number">1.1</span> Need for better therapeutics</a></li>
<li><a href="1-2-engineered-human-myocardium.html#engineered-human-myocardium"><span class="toc-section-number">1.2</span> Engineered Human Myocardium</a></li>
<li class="has-sub"><a href="1-3-rna-sequencing.html#rna-sequencing"><span class="toc-section-number">1.3</span> RNA Sequencing</a><ul>
<li><a href="1-3-rna-sequencing.html#bulk-rna-seq"><span class="toc-section-number">1.3.1</span> Bulk RNA Seq</a></li>
<li><a href="1-3-rna-sequencing.html#single-cell-rna-seq"><span class="toc-section-number">1.3.2</span> Single-cell RNA Seq</a></li>
</ul></li>
<li><a href="1-4-computational-deconvolution.html#computational-deconvolution"><span class="toc-section-number">1.4</span> Computational deconvolution</a></li>
<li><a href="1-5-principal-component-analysis-pca.html#principal-component-analysis-pca"><span class="toc-section-number">1.5</span> Principal Component Analysis (PCA)</a></li>
<li><a href="1-6-rationale-for-the-current-work.html#rationale-for-the-current-work"><span class="toc-section-number">1.6</span> Rationale for the current work</a></li>
</ul></li>
<li><a href="2-aims-and-objectives.html#aims-and-objectives"><span class="toc-section-number">2</span> Aims and Objectives</a></li>
<li><a href="3-materials-and-methods.html#materials-and-methods"><span class="toc-section-number">3</span> Materials and Methods</a></li>
<li><a href="4-results-and-discussion.html#results-and-discussion"><span class="toc-section-number">4</span> Results and Discussion</a></li>
<li><a href="5-conclusion-and-future-work.html#conclusion-and-future-work"><span class="toc-section-number">5</span> Conclusion and Future Work</a></li>
<li><a href="6-references.html#references"><span class="toc-section-number">6</span> References</a></li>
</ul>
</div>
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="computational-deconvolution" class="section level2">
<h2><span class="header-section-number">1.4</span> Computational deconvolution</h2>
<p>It is established that scRNA-Seq allows for unbiased transcriptional profiling of thousands of individual cells. Yet, the usage of this powerful technology is limited by its cost and impracticality with respect to analyses of large sample cohorts. Also, most clinical specimens are fixed, for example in formalin or embedded in paraffin, which renders its dissociation into intact single-cells impossible. To circumvent these limitations and utilize the specificity and accuracy of scRNA-Seq along with the ease of bulk of RNA-Seq, several groups around the world have developed <em>deconvolution</em> computational techniques <span class="citation">(Aran, Hu, and Butte <a href="#ref-aranXCellDigitallyPortraying2017">2017</a>; Becht et al. <a href="#ref-bechtEstimatingPopulationAbundance2016">2016</a>; Dong et al., <a href="#ref-dongSCDCBulkGene">n.d.</a>; Kang et al. <a href="#ref-kangCDSeqNovelComplete2019">2019</a>; Newman and Alizadeh <a href="#ref-newmanHighthroughputGenomicProfiling2016">2016</a>; Quon et al. <a href="#ref-quonComputationalPurificationIndividual2013">2013</a>; Racle et al. <a href="#ref-racleSimultaneousEnumerationCancer2017">2017</a>; Shen-Orr and Gaujoux <a href="#ref-shen-orrComputationalDeconvolutionExtracting2013">2013</a>)</span>. Deconvolution, in the realm of a sequencing, is a common umbrella term for a procedure that estimates the proportion of each cell type in a bulk sample. Flow cytometry and scRNA-Seq are experimental methods of deconvolution. Computational deconvolution leverages scRNA-Seq reference sets (or fluorescence-activated cell sorting (FACS)-sorted, purified bulk sets) for bulk gene expression deconvolution. A basic comparison of the available tools soon led to CIBERSORTx (<span class="citation">(Newman et al. <a href="#ref-newmanDeterminingCellType2019">2019</a>)</span>) as being currently at the forefront of deconvolution because unlike other methods it can:
1.Leverage scRNA-Seq derived reference profiled for bulk tissue dissection
2. Overcome technical variation arising from different platforms (eg., bulk RNA-Seq, scRNA-Seq, microarrays) and tissue preservation techniques
3. Digitally “purify” cell-type specific expression profiles from bulk tissues without physical cell isolation.
Briefly, most deconvolution algorithms, including CIBERSORTx, work to solve the following linear equations for <strong>f</strong>:
<span class="math display">\[m = Hf\]</span>
<em>m</em>: mixture gene expression profile (GEP) (to be deconvolved)</p>
<p><em>f</em>: a vector of fraction of each cell type in a signature matrix (the unknown)</p>
<p><em>H</em>: a <em>signature matrix</em> containing signature genes for cell subsets of interest</p>
Both <em>m</em> and <em>B</em> are input requirements. Further analytics of deconvolution are outside the scope of this thesis and be found at <span class="citation">(Chen et al. <a href="#ref-chenProfilingTumorInfiltrating2018">2018</a>; Newman et al. <a href="#ref-newmanDeterminingCellType2019">2019</a>)</span>. So with this framework, a relevant single-cell or bulk-sorted RNA sequencing data can be used to tease out molecular signatures of distinct cell types and these signatures can then be used to characterize cellular heterogeneity from bulk tissue transcriptomes without physical cell isolation, see <a href="1-4-computational-deconvolution.html#fig:deconv">1.3</a>.
<div class="figure"><span id="fig:deconv"></span>
<img src="data/deconv.png" alt="A caption" width="100%" />
<p class="caption">
Figure 1.3: A caption
</p>
</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-aranXCellDigitallyPortraying2017">
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: Digitally Portraying the Tissue Cellular Heterogeneity Landscape.” <em>Genome Biology</em> 18 (1): 220. <a href="https://doi.org/10.1186/s13059-017-1349-1">https://doi.org/10.1186/s13059-017-1349-1</a>.</p>
</div>
<div id="ref-bechtEstimatingPopulationAbundance2016">
<p>Becht, Etienne, Nicolas A. Giraldo, Laetitia Lacroix, B’en’edicte Buttard, Nabila Elarouci, Florent Petitprez, Janick Selves, et al. 2016. “Estimating the Population Abundance of Tissue-Infiltrating Immune and Stromal Cell Populations Using Gene Expression.” <em>Genome Biology</em> 17 (1): 218. <a href="https://doi.org/10.1186/s13059-016-1070-5">https://doi.org/10.1186/s13059-016-1070-5</a>.</p>
</div>
<div id="ref-chenProfilingTumorInfiltrating2018">
<p>Chen, Binbin, Michael S. Khodadoust, Chih Long Liu, Aaron M. Newman, and Ash A. Alizadeh. 2018. “Profiling Tumor Infiltrating Immune Cells with CIBERSORT.” <em>Methods in Molecular Biology (Clifton, N.J.)</em> 1711: 243–59. <a href="https://doi.org/10.1007/978-1-4939-7493-1_12">https://doi.org/10.1007/978-1-4939-7493-1_12</a>.</p>
</div>
<div id="ref-dongSCDCBulkGene">
<p>Dong, Meichen, Aatish Thennavan, Eugene Urrutia, Yun Li, Charles M. Perou, Fei Zou, and Yuchao Jiang. n.d. “SCDC: Bulk Gene Expression Deconvolution by Multiple Single-Cell RNA Sequencing References.” <em>Briefings in Bioinformatics</em>. <a href="https://doi.org/10.1093/bib/bbz166">https://doi.org/10.1093/bib/bbz166</a>.</p>
</div>
<div id="ref-kangCDSeqNovelComplete2019">
<p>Kang, Kai, Qian Meng, Igor Shats, David M. Umbach, Melissa Li, Yuanyuan Li, Xiaoling Li, and Leping Li. 2019. “CDSeq: A Novel Complete Deconvolution Method for Dissecting Heterogeneous Samples Using Gene Expression Data.” <em>PLOS Computational Biology</em> 15 (12): e1007510. <a href="https://doi.org/10.1371/journal.pcbi.1007510">https://doi.org/10.1371/journal.pcbi.1007510</a>.</p>
</div>
<div id="ref-newmanHighthroughputGenomicProfiling2016">
<p>Newman, Aaron M., and Ash A. Alizadeh. 2016. “High-Throughput Genomic Profiling of Tumor-Infiltrating Leukocytes.” <em>Current Opinion in Immunology</em> 41 (August): 77–84. <a href="https://doi.org/10.1016/j.coi.2016.06.006">https://doi.org/10.1016/j.coi.2016.06.006</a>.</p>
</div>
<div id="ref-newmanDeterminingCellType2019">
<p>Newman, Aaron M., Chlo’e B. Steen, Chih Long Liu, Andrew J. Gentles, Aadel A. Chaudhuri, Florian Scherer, Michael S. Khodadoust, et al. 2019. “Determining Cell Type Abundance and Expression from Bulk Tissues with Digital Cytometry.” <em>Nature Biotechnology</em> 37 (7): 773–82. <a href="https://doi.org/10.1038/s41587-019-0114-2">https://doi.org/10.1038/s41587-019-0114-2</a>.</p>
</div>
<div id="ref-quonComputationalPurificationIndividual2013">
<p>Quon, Gerald, Syed Haider, Amit G. Deshwar, Ang Cui, Paul C. Boutros, and Quaid Morris. 2013. “Computational Purification of Individual Tumor Gene Expression Profiles Leads to Significant Improvements in Prognostic Prediction.” <em>Genome Medicine</em> 5 (3): 29. <a href="https://doi.org/10.1186/gm433">https://doi.org/10.1186/gm433</a>.</p>
</div>
<div id="ref-racleSimultaneousEnumerationCancer2017">
<p>Racle, Julien, Kaat de Jonge, Petra Baumgaertner, Daniel E. Speiser, and David Gfeller. 2017. “Simultaneous Enumeration of Cancer and Immune Cell Types from Bulk Tumor Gene Expression Data.” <em>eLife</em>. https://elifesciences.org/articles/26476. <a href="https://doi.org/10.7554/eLife.26476">https://doi.org/10.7554/eLife.26476</a>.</p>
</div>
<div id="ref-shen-orrComputationalDeconvolutionExtracting2013">
<p>Shen-Orr, Shai S, and Renaud Gaujoux. 2013. “Computational Deconvolution: Extracting Cell Type-Specific Information from Heterogeneous Samples.” <em>Current Opinion in Immunology</em>, Special section: Systems biology and bioinformatics / Immunogenetics and transplantation, 25 (5): 571–78. <a href="https://doi.org/10.1016/j.coi.2013.09.015">https://doi.org/10.1016/j.coi.2013.09.015</a>.</p>
</div>
</div>
<p style="text-align: center;">
<a href="1-3-rna-sequencing.html"><button class="btn btn-default">Previous</button></a>
<a href="1-5-principal-component-analysis-pca.html"><button class="btn btn-default">Next</button></a>
</p>
</div>
</div>


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
