<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | RNA-Sequencing to improve characterisation and production of iPSC-induced cardiomyocytes</title>
  <meta name="description" content="References | RNA-Sequencing to improve characterisation and production of iPSC-induced cardiomyocytes" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="References | RNA-Sequencing to improve characterisation and production of iPSC-induced cardiomyocytes" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | RNA-Sequencing to improve characterisation and production of iPSC-induced cardiomyocytes" />
  
  
  

<meta name="author" content="Harithaa Anandakumar" />


<meta name="date" content="2020-01-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="summary.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.46.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.46.1/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet" />



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#need-for-better-therapeutics"><i class="fa fa-check"></i><b>1.1</b> Need for better therapeutics</a><ul>
<li class="chapter" data-level="1.1.1" data-path="index.html"><a href="index.html#immunological-responses-in-transplantations"><i class="fa fa-check"></i><b>1.1.1</b> Immunological Responses in Transplantations</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#engineered-human-myocardium"><i class="fa fa-check"></i><b>1.2</b> Engineered Human Myocardium</a><ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#cgmp-and-quality-control-of-tissue-engineered-products"><i class="fa fa-check"></i><b>1.2.1</b> cGMP and Quality Control of Tissue Engineered Products</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#rnaseq"><i class="fa fa-check"></i><b>1.3</b> RNA Sequencing</a><ul>
<li class="chapter" data-level="1.3.1" data-path="index.html"><a href="index.html#bulkrna"><i class="fa fa-check"></i><b>1.3.1</b> Single cell versus bulk RNA Seq</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#compdeconv"><i class="fa fa-check"></i><b>1.4</b> Computational deconvolution</a></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#exploratory-data-analysis-in-rna-sequencing"><i class="fa fa-check"></i><b>1.5</b> Exploratory Data Analysis in RNA-Sequencing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="index.html"><a href="index.html#pca"><i class="fa fa-check"></i><b>1.5.1</b> Principal Component Analysis (PCA)</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="index.html"><a href="index.html#rationale"><i class="fa fa-check"></i><b>1.6</b> Rationale for the current work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="aims-and-objectives.html"><a href="aims-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Aims and Objectives</a></li>
<li class="chapter" data-level="3" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>3</b> Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="methods.html"><a href="methods.html#general-analysis-pipeline-of-bulk-rna-seq-data"><i class="fa fa-check"></i><b>3.1</b> General Analysis Pipeline of Bulk RNA-Seq Data</a></li>
<li class="chapter" data-level="3.2" data-path="methods.html"><a href="methods.html#singleCell"><i class="fa fa-check"></i><b>3.2</b> Single Cell Reference Data and CIBERSORTX</a><ul>
<li class="chapter" data-level="3.2.1" data-path="methods.html"><a href="methods.html#processing-of-single-cell-data"><i class="fa fa-check"></i><b>3.2.1</b> Processing of Single Cell Data</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#analysis-of-rhesus-rna-seq"><i class="fa fa-check"></i><b>3.3</b> Analysis of Rhesus RNA-Seq</a></li>
<li class="chapter" data-level="3.4" data-path="methods.html"><a href="methods.html#estBacVir"><i class="fa fa-check"></i><b>3.4</b> Estimating Bacterial and Viral Contaminants</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results-and-discussion.html"><a href="results-and-discussion.html"><i class="fa fa-check"></i><b>4</b> Results and Discussion</a><ul>
<li class="chapter" data-level="4.1" data-path="results-and-discussion.html"><a href="results-and-discussion.html#general-workflow-and-mapping-statistics"><i class="fa fa-check"></i><b>4.1</b> General Workflow and Mapping Statistics</a></li>
<li class="chapter" data-level="4.2" data-path="results-and-discussion.html"><a href="results-and-discussion.html#exploring-potential-microbial-contamination-using-rna-seq-data"><i class="fa fa-check"></i><b>4.2</b> Exploring Potential Microbial Contamination using RNA-Seq Data</a></li>
<li class="chapter" data-level="4.3" data-path="results-and-discussion.html"><a href="results-and-discussion.html#global-view-of-the-transcriptomic-data"><i class="fa fa-check"></i><b>4.3</b> Global view of the transcriptomic data</a><ul>
<li class="chapter" data-level="4.3.1" data-path="results-and-discussion.html"><a href="results-and-discussion.html#correlation-amongst-groups"><i class="fa fa-check"></i><b>4.3.1</b> Correlation amongst groups</a></li>
<li class="chapter" data-level="4.3.2" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gene-level-analysis"><i class="fa fa-check"></i><b>4.3.2</b> Gene-level analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="results-and-discussion.html"><a href="results-and-discussion.html#deconvolution-of-bulk-cms-and-ehms-rna-seq-data"><i class="fa fa-check"></i><b>4.4</b> Deconvolution of Bulk CMs and EHMs RNA-Seq Data</a><ul>
<li class="chapter" data-level="4.4.1" data-path="results-and-discussion.html"><a href="results-and-discussion.html#limits-of-deconvolution"><i class="fa fa-check"></i><b>4.4.1</b> Limits of deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="results-and-discussion.html"><a href="results-and-discussion.html#basic-characterisation-of-rhesus-cardiomyocytes"><i class="fa fa-check"></i><b>4.5</b> Basic characterisation of Rhesus Cardiomyocytes</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="conclusion-and-future-work.html"><a href="conclusion-and-future-work.html"><i class="fa fa-check"></i><b>5</b> Conclusion and Future Work</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a><ul>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html#task-at-hand"><i class="fa fa-check"></i>Task At Hand</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html#work-done"><i class="fa fa-check"></i>Work Done</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">RNA-Sequencing to improve characterisation and production of iPSC-induced cardiomyocytes</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>1. Max Roser, E. O.-O. &amp; Ritchie, H. Life expectancy. <em>Our World in Data</em> (2020).</p>
</div>
<div>
<p>2. Ritchie, H. &amp; Roser, M. Causes of death. <em>Our World in Data</em> (2020).</p>
</div>
<div>
<p>3. Taylor, C. J. <em>et al.</em> Trends in survival after a diagnosis of heart failure in the United Kingdom 2000-2017: Population based cohort study. <em>BMJ</em> <strong>364</strong>, (2019).</p>
</div>
<div>
<p>4. Liao, L., Allen, L. A. &amp; Whellan, D. J. Economic burden of heart failure in the elderly. <em>PharmacoEconomics</em> <strong>26</strong>, 447–462 (2008).</p>
</div>
<div>
<p>5. Cook, C., Cole, G., Asaria, P., Jabbour, R. &amp; Francis, D. P. The annual global economic burden of heart failure. <em>International Journal of Cardiology</em> <strong>171</strong>, 368–376 (2014).</p>
</div>
<div>
<p>6. Lesyuk, W., Kriza, C. &amp; Kolominsky-Rabas, P. Cost-of-illness studies in heart failure: A systematic review 20042016. <em>BMC Cardiovascular Disorders</em> <strong>18</strong>, 74 (2018).</p>
</div>
<div>
<p>7. Trivedi, J. R. <em>et al.</em> (574) - Risk Factors of Waiting List Mortality for Patients Awaiting Heart Transplant. <em>The Journal of Heart and Lung Transplantation</em> <strong>35</strong>, S214 (2016).</p>
</div>
<div>
<p>8. Eurotransplant - Statistics.</p>
</div>
<div>
<p>9. Metra, M. &amp; Teerlink, J. R. Heart failure. <em>The Lancet</em> <strong>390</strong>, 1981–1995 (2017).</p>
</div>
<div>
<p>10. Bergmann, O. <em>et al.</em> Dynamics of Cell Generation and Turnover in the Human Heart. <em>Cell</em> <strong>161</strong>, 1566–1575 (2015).</p>
</div>
<div>
<p>11. Thomson, J. A. <em>et al.</em> Embryonic stem cell lines derived from human blastocysts. <em>Science (New York, N.Y.)</em> <strong>282</strong>, 1145–1147 (1998).</p>
</div>
<div>
<p>12. Takahashi, K. <em>et al.</em> Induction of pluripotent stem cells from adult human fibroblasts by defined factors. <em>Cell</em> <strong>131</strong>, 861–872 (2007).</p>
</div>
<div>
<p>13. Burridge, P. W., Keller, G., Gold, J. D. &amp; Wu, J. C. Production of de novo cardiomyocytes: Human pluripotent stem cell differentiation and direct reprogramming. <em>Cell Stem Cell</em> <strong>10</strong>, 16–28 (2012).</p>
</div>
<div>
<p>14. Nguyen, P. K., Neofytou, E., Rhee, J.-W. &amp; Wu, J. C. Potential Strategies to Address the Major Clinical Barriers Facing Stem Cell Regenerative Therapy for Cardiovascular Disease: A Review. <em>JAMA cardiology</em> <strong>1</strong>, 953–962 (2016).</p>
</div>
<div>
<p>15. Inagawa, K. &amp; Ieda, M. Direct reprogramming of mouse fibroblasts into cardiac myocytes. <em>Journal of Cardiovascular Translational Research</em> <strong>6</strong>, 37–45 (2013).</p>
</div>
<div>
<p>16. Kubin, T. <em>et al.</em> Oncostatin M is a major mediator of cardiomyocyte dedifferentiation and remodeling. <em>Cell Stem Cell</em> <strong>9</strong>, 420–432 (2011).</p>
</div>
<div>
<p>17. Gnecchi, M. <em>et al.</em> Paracrine action accounts for marked protection of ischemic heart by Akt-modified mesenchymal stem cells. <em>Nature Medicine</em> <strong>11</strong>, 367–368 (2005).</p>
</div>
<div>
<p>18. Sekine, H. <em>et al.</em> Cardiac cell sheet transplantation improves damaged heart function via superior cell survival in comparison with dissociated cell injection. <em>Tissue Engineering. Part A</em> <strong>17</strong>, 2973–2980 (2011).</p>
</div>
<div>
<p>19. Weinberger, F. <em>et al.</em> Cardiac repair in guinea pigs with human engineered heart tissue from induced pluripotent stem cells. <em>Science Translational Medicine</em> <strong>8</strong>, 363ra148 (2016).</p>
</div>
<div>
<p>20. Yang, T. <em>et al.</em> Cardiac engraftment of genetically-selected parthenogenetic stem cell-derived cardiomyocytes. <em>PloS One</em> <strong>10</strong>, e0131511 (2015).</p>
</div>
<div>
<p>21. Zimmermann, W.-H. <em>et al.</em> Engineered heart tissue grafts improve systolic and diastolic function in infarcted rat hearts. <em>Nature Medicine</em> <strong>12</strong>, 452–458 (2006).</p>
</div>
<div>
<p>22. Liu, Y.-W. <em>et al.</em> Human embryonic stem cell-derived cardiomyocytes restore function in infarcted hearts of non-human primates. <em>Nature Biotechnology</em> <strong>36</strong>, 597–605 (2018).</p>
</div>
<div>
<p>23. Neofytou, E., O’Brien, C. G., Couture, L. A. &amp; Wu, J. C. Hurdles to clinical translation of human induced pluripotent stem cells. <em>The Journal of Clinical Investigation</em> <strong>125</strong>, 2551–2557 (2015).</p>
</div>
<div>
<p>24. Sayed, N., Liu, C. &amp; Wu, J. C. Translation of Human-Induced Pluripotent Stem Cells: From Clinical Trial in a Dish to Precision Medicine. <em>Journal of the American College of Cardiology</em> <strong>67</strong>, 2161–2176 (2016).</p>
</div>
<div>
<p>25. Martin, U. Therapeutic Application of Pluripotent Stem Cells: Challenges and Risks. <em>Frontiers in Medicine</em> <strong>4</strong>, (2017).</p>
</div>
<div>
<p>26. Taylor, C. J. <em>et al.</em> Banking on human embryonic stem cells: Estimating the number of donor cell lines needed for HLA matching. <em>The Lancet</em> <strong>366</strong>, 2019–2025 (2005).</p>
</div>
<div>
<p>27. Nakatsuji, N., Nakajima, F. &amp; Tokunaga, K. HLA-haplotype banking and iPS cells. <em>Nature Biotechnology</em> <strong>26</strong>, 739–740 (2008).</p>
</div>
<div>
<p>28. Bogomiakova, M. E., Eremeev, A. V. &amp; Lagarkova, M. A. At Home among Strangers: Is It Possible to Create Hypoimmunogenic Pluripotent Stem Cell Lines? <em>Molecular Biology</em> <strong>53</strong>, 638–652 (2019).</p>
</div>
<div>
<p>29. Han, X. <em>et al.</em> Generation of hypoimmunogenic human pluripotent stem cells. <em>Proceedings of the National Academy of Sciences</em> <strong>116</strong>, 10441–10446 (2019).</p>
</div>
<div>
<p>30. Tiburcy, M. <em>et al.</em> Defined Engineered Human Myocardium with Advanced Maturation for Applications in Heart Failure Modelling and Repair. <em>Circulation</em> <strong>135</strong>, 1832–1847 (2017).</p>
</div>
<div>
<p>31. 2006, E. EMEA/chmp 2006. Guideline on human cell-based medicinal products. EMEA/chmp/410869/2006. (2006).</p>
</div>
<div>
<p>32. CD-P-TO. Guideline to the quality and safety of tissues and cells for human applications. <em>European Committee on Organ Transplantation,EDQM</em> (2017).</p>
</div>
<div>
<p>33. Wang, Z., Gerstein, M. &amp; Snyder, M. RNA-Seq: A revolutionary tool for transcriptomics. <em>Nature reviews. Genetics</em> <strong>10</strong>, 57–63 (2009).</p>
</div>
<div>
<p>34. Conesa, A. <em>et al.</em> A survey of best practices for RNA-seq data analysis. <em>Genome Biology</em> <strong>17</strong>, 13 (2016).</p>
</div>
<div>
<p>35. Tang, F. <em>et al.</em> mRNA-Seq whole-transcriptome analysis of a single cell. <em>Nature Methods</em> <strong>6</strong>, 377–382 (2009).</p>
</div>
<div>
<p>36. Montoro, D. T. <em>et al.</em> A revised airway epithelial hierarchy includes CFTR-expressing ionocytes. <em>Nature</em> <strong>560</strong>, 319–324 (2018).</p>
</div>
<div>
<p>37. Asp, M. <em>et al.</em> Spatial detection of fetal marker genes expressed at low level in adult human heart tissue. <em>Scientific Reports</em> <strong>7</strong>, (2017).</p>
</div>
<div>
<p>38. Stegle, O., Teichmann, S. A. &amp; Marioni, J. C. Computational and analytical challenges in single-cell transcriptomics. <em>Nature Reviews. Genetics</em> <strong>16</strong>, 133–145 (2015).</p>
</div>
<div>
<p>39. Svensson, V., Vento-Tormo, R. &amp; Teichmann, S. A. Exponential scaling of single-cell RNA-seq in the past decade. <em>Nature Protocols</em> <strong>13</strong>, 599–604 (2018).</p>
</div>
<div>
<p>40. Aran, D., Hu, Z. &amp; Butte, A. J. xCell: Digitally portraying the tissue cellular heterogeneity landscape. <em>Genome Biology</em> <strong>18</strong>, 220 (2017).</p>
</div>
<div>
<p>41. Becht, E. <em>et al.</em> Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression. <em>Genome Biology</em> <strong>17</strong>, 218 (2016).</p>
</div>
<div>
<p>42. Kang, K. <em>et al.</em> CDSeq: A novel complete deconvolution method for dissecting heterogeneous samples using gene expression data. <em>PLOS Computational Biology</em> <strong>15</strong>, e1007510 (2019).</p>
</div>
<div>
<p>43. Newman, A. M. &amp; Alizadeh, A. A. High-throughput genomic profiling of tumor-infiltrating leukocytes. <em>Current Opinion in Immunology</em> <strong>41</strong>, 77–84 (2016).</p>
</div>
<div>
<p>44. Quon, G. <em>et al.</em> Computational purification of individual tumor gene expression profiles leads to significant improvements in prognostic prediction. <em>Genome Medicine</em> <strong>5</strong>, 29 (2013).</p>
</div>
<div>
<p>45. Racle, J., de Jonge, K., Baumgaertner, P., Speiser, D. E. &amp; Gfeller, D. Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data. <em>eLife</em> (2017) doi:<a href="https://doi.org/10.7554/eLife.26476">10.7554/eLife.26476</a>.</p>
</div>
<div>
<p>46. Shen-Orr, S. S. &amp; Gaujoux, R. Computational deconvolution: Extracting cell type-specific information from heterogeneous samples. <em>Current Opinion in Immunology</em> <strong>25</strong>, 571–578 (2013).</p>
</div>
<div>
<p>47. Newman, A. M. <em>et al.</em> Determining cell type abundance and expression from bulk tissues with digital cytometry. <em>Nature Biotechnology</em> <strong>37</strong>, 773–782 (2019).</p>
</div>
<div>
<p>48. Chen, B., Khodadoust, M. S., Liu, C. L., Newman, A. M. &amp; Alizadeh, A. A. Profiling tumor infiltrating immune cells with CIBERSORT. <em>Methods in molecular biology (Clifton, N.J.)</em> <strong>1711</strong>, 243–259 (2018).</p>
</div>
<div>
<p>49. Hudson, N. J., Dalrymple, B. P. &amp; Reverter, A. Beyond differential expression: The quest for causal mutations and effector molecules. <em>BMC Genomics</em> <strong>13</strong>, 356 (2012).</p>
</div>
<div>
<p>50. Witteveen, E. <em>et al.</em> Increased Early Systemic Inflammation in ICU-Acquired Weakness; A Prospective Observational Cohort Study*. <em>Critical Care Medicine</em> <strong>45</strong>, 972–979 (2017).</p>
</div>
<div>
<p>51. Cuomo, A. S. E. <em>et al.</em> Single-cell RNA-sequencing of differentiating iPS cells reveals dynamic genetic effects on gene expression. <em>Nature Communications</em> <strong>11</strong>, 1–14 (2020).</p>
</div>
<div>
<p>52. Han, X. <em>et al.</em> Mapping human pluripotent stem cell differentiation pathways using high throughput single-cell RNA-sequencing. <em>Genome Biology</em> <strong>19</strong>, 47 (2018).</p>
</div>
<div>
<p>53. McCracken, I. <em>et al.</em> Transcriptional dynamics of pluripotent stem cell-derived endothelial cell differentiation revealed by single-cell RNA sequencing. <em>Eur Heart J</em> (2019) doi:<a href="https://doi.org/10.1093/eurheartj/ehz351">10.1093/eurheartj/ehz351</a>.</p>
</div>
<div>
<p>54. Müller, G. A., Tarasov, K. V., Gundry, R. L. &amp; Boheler, K. R. Human ESC/iPSC-based ‘omics’ and bioinformatics for translational research. <em>Drug Discovery Today: Disease Models</em> <strong>9</strong>, e161–e170 (2012).</p>
</div>
<div>
<p>55. Wesolowska-Andersen, A. <em>et al.</em> Analysis of Differentiation Protocols Defines a Common Pancreatic Progenitor Molecular Signature and Guides Refinement of Endocrine Differentiation. <em>Stem Cell Reports</em> <strong>14</strong>, 138–153 (2020).</p>
</div>
<div>
<p>56. Wu, H. <em>et al.</em> Comparative Analysis and Refinement of Human PSC-Derived Kidney Organoid Differentiation with Single-Cell Transcriptomics. <em>Cell Stem Cell</em> <strong>23</strong>, 869–881.e8 (2018).</p>
</div>
<div>
<p>57. Freedman, B. S. Better Being Single? Omics Improves Kidney Organoids. <em>Nephron</em> <strong>141</strong>, 128–132 (2019).</p>
</div>
<div>
<p>58. Friedman, C. E. <em>et al.</em> Single-Cell Transcriptomic Analysis of Cardiac Differentiation from Human PSCs Reveals HOPX-Dependent Cardiomyocyte Maturation. <em>Cell Stem Cell</em> <strong>23</strong>, 586–598.e8 (2018).</p>
</div>
<div>
<p>59. Kuppusamy, K. T. <em>et al.</em> Let-7 family of microRNA is required for maturation and adult-like metabolism in stem cell-derived cardiomyocytes. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>112</strong>, E2785–2794 (2015).</p>
</div>
<div>
<p>60. Mills, R. J. <em>et al.</em> Functional screening in human cardiac organoids reveals a metabolic mechanism for cardiomyocyte cell cycle arrest. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>114</strong>, E8372–E8381 (2017).</p>
</div>
<div>
<p>61. Pavlovic, B. J., Blake, L. E., Roux, J., Chavarria, C. &amp; Gilad, Y. A Comparative Assessment of Human and Chimpanzee iPSC-derived Cardiomyocytes with Primary Heart Tissues. <em>Scientific Reports</em> <strong>8</strong>, 15312 (2018).</p>
</div>
<div>
<p>62. Pervolaraki, E., Dachtler, J., Anderson, R. A. &amp; Holden, A. V. The developmental transcriptome of the human heart. <em>Scientific Reports</em> <strong>8</strong>, (2018).</p>
</div>
<div>
<p>63. Yan, L. <em>et al.</em> Epigenomic Landscape of Human Fetal Brain, Heart, and Liver. <em>The Journal of Biological Chemistry</em> <strong>291</strong>, 4386–4398 (2016).</p>
</div>
<div>
<p>64. Sangiovanni, M., Granata, I., Thind, A. S. &amp; Guarracino, M. R. From trash to treasure: Detecting unexpected contamination in unmapped NGS data. <em>BMC Bioinformatics</em> <strong>20</strong>, 168 (2019).</p>
</div>
<div>
<p>65. Schaffer, J. N. &amp; Pearson, M. M. Proteus mirabilis and Urinary Tract Infections. <em>Microbiology spectrum</em> <strong>3</strong>, (2015).</p>
</div>
<div>
<p>66. Drzewiecka, D. Significance and Roles of Proteus spp. Bacteria in Natural Environments. <em>Microbial Ecology</em> <strong>72</strong>, 741–758 (2016).</p>
</div>
<div>
<p>67. Grandi, N. &amp; Tramontano, E. Human Endogenous Retroviruses Are Ancient Acquired Elements Still Shaping Innate Immune Responses. <em>Frontiers in Immunology</em> <strong>9</strong>, (2018).</p>
</div>
<div>
<p>68. Küry, P. <em>et al.</em> Human Endogenous Retroviruses in Neurological Diseases. <em>Trends in Molecular Medicine</em> <strong>24</strong>, 379–394 (2018).</p>
</div>
<div>
<p>69. Nelson, P. N. <em>et al.</em> Demystified . . . Human endogenous retroviruses. <em>Molecular Pathology</em> <strong>56</strong>, 11–18 (2003).</p>
</div>
<div>
<p>70. Strong, M. J. <em>et al.</em> Microbial Contamination in Next Generation Sequencing: Implications for Sequence-Based Analysis of Clinical Samples. <em>PLoS Pathogens</em> <strong>10</strong>, (2014).</p>
</div>
<div>
<p>71. K’eki, Z., Gr’ebner, K., Bohus, V., M’arialigeti, K. &amp; T’oth, E. M. Application of special oligotrophic media for cultivation of bacterial communities originated from ultrapure water. <em>Acta Microbiologica Et Immunologica Hungarica</em> <strong>60</strong>, 345–357 (2013).</p>
</div>
<div>
<p>72. Kulakov, L. A., McAlister, M. B., Ogden, K. L., Larkin, M. J. &amp; O’Hanlon, J. F. Analysis of bacteria contaminating ultrapure water in industrial systems. <em>Applied and Environmental Microbiology</em> <strong>68</strong>, 1548–1555 (2002).</p>
</div>
<div>
<p>73. Salter, S. J. <em>et al.</em> Reagent and laboratory contamination can critically impact sequence-based microbiome analyses. <em>BMC Biology</em> <strong>12</strong>, 87 (2014).</p>
</div>
<div>
<p>74. Bengoechea, J. A. &amp; Sa Pessoa, J. Klebsiella pneumoniae infection biology: Living to counteract host defences. <em>FEMS Microbiology Reviews</em> <strong>43</strong>, 123–144 (2019).</p>
</div>
<div>
<p>75. Escobar, A., Rodas, P. I. &amp; Acuña-Castillo, C. MacrophageNeisseria gonorrhoeae Interactions: A Better Understanding of Pathogen Mechanisms of Immunomodulation. <em>Frontiers in Immunology</em> <strong>9</strong>, (2018).</p>
</div>
<div>
<p>76. Park, S.-J. <em>et al.</em> A systematic sequencing-based approach for microbial contaminant detection and functional inference. <em>BMC Biology</em> <strong>17</strong>, 72 (2019).</p>
</div>
<div>
<p>77. Madeira, A., Camps, M., Zorzano, A., Moura, T. F. &amp; Soveral, G. Biophysical Assessment of Human Aquaporin-7 as a Water and Glycerol Channel in 3T3-L1 Adipocytes. <em>PLOS ONE</em> <strong>8</strong>, e83442 (2013).</p>
</div>
<div>
<p>78. Nordquist, E., LaHaye, S., Nagel, C. &amp; Lincoln, J. Postnatal and Adult Aortic Heart Valves Have Distinctive Transcriptional Profiles Associated With Valve Tissue Growth and Maintenance Respectively. <em>Frontiers in Cardiovascular Medicine</em> <strong>5</strong>, (2018).</p>
</div>
<div>
<p>79. Tiburcy Malte <em>et al.</em> Defined Engineered Human Myocardium With Advanced Maturation for Applications in Heart Failure Modeling and Repair. <em>Circulation</em> <strong>135</strong>, 1832–1847 (2017).</p>
</div>
<div>
<p>80. Cai Wenxuan <em>et al.</em> An Unbiased Proteomics Method to Assess the Maturation of Human Pluripotent Stem CellDerived Cardiomyocytes. <em>Circulation Research</em> <strong>125</strong>, 936–953 (2019).</p>
</div>
<div>
<p>81. Fougerousse, F. <em>et al.</em> Calpain3 expression during human cardiogenesis. <em>Neuromuscular disorders: NMD</em> <strong>10</strong>, 251–256 (2000).</p>
</div>
<div>
<p>82. Liu Qing <em>et al.</em> Genome-Wide Temporal Profiling of Transcriptome and Open Chromatin of Early Cardiomyocyte Differentiation Derived From hiPSCs and hESCs. <em>Circulation Research</em> <strong>121</strong>, 376–391 (2017).</p>
</div>
<div>
<p>83. Ng, A. <em>et al.</em> Loss of glypican-3 Function Causes Growth Factor-dependent Defects in Cardiac and Coronary Vascular Development. <em>Developmental biology</em> <strong>335</strong>, 208–215 (2009).</p>
</div>
<div>
<p>84. Pawlak, M. <em>et al.</em> Dynamics of cardiomyocyte transcriptome and chromatin landscape demarcates key events of heart development. <em>Genome Research</em> <strong>29</strong>, 506–519 (2019).</p>
</div>
<div>
<p>85. Schiaffino, S., Rossi, A. C., Smerdu, V., Leinwand, L. A. &amp; Reggiani, C. Developmental myosins: Expression patterns and functional significance. <em>Skeletal Muscle</em> <strong>5</strong>, 22 (2015).</p>
</div>
<div>
<p>86. Wang, T. Y. <em>et al.</em> Human cardiac myosin light chain 4 (MYL4) mosaic expression patterns vary by sex. <em>Scientific Reports</em> <strong>9</strong>, 1–7 (2019).</p>
</div>
<div>
<p>87. Sheng, J.-J. &amp; Jin, J.-P. TNNI1, TNNI2 and TNNI3: Evolution, Regulation, and Protein Structure-Function Relationships. <em>Gene</em> <strong>576</strong>, 385–394 (2016).</p>
</div>
<div>
<p>88. Machiraju, P. &amp; Greenway, S. C. Current methods for the maturation of induced pluripotent stem cell-derived cardiomyocytes. <em>World Journal of Stem Cells</em> <strong>11</strong>, 33–43 (2019).</p>
</div>
</div>
</div>









            </section>

          </div>
        </div>
      </div>
<a href="summary.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
